.github/ISSUE_TEMPLATE/refactor_request.yaml

```diff
@@ -0,0 +1,26 @@
+name: Refactor Request
+description: Suggest something that needs to be refactored
+title: "[Refactor]: "
+labels: [refactor]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for the feature request!
+  - type: textarea
+    id: request
+    attributes:
+      label: Refactor Request
+      description: Is your refactor request related to a problem? Please describe.
+      placeholder: A clear and concise description of what the problem is. Ex. I'm having problems reading the code because [...] (nice to have file/line references too)
+      value: ""
```

Comment:
I agree not setting a default value, it was a bit annoying removing it every time (although looks like it's getting a warning of some sort)
![image](https://user-images.githubusercontent.com/34047281/169705542-1201390b-14b3-4386-aad9-86bdef2c4879.png)

Reply:
how did you get this warning??

Reply:
I think I looked at the file from the branch (not the pr)